hlx_statics/styles/styles.css

```diff
@@ -11,6 +11,7 @@
  */
 
  @import url('./spectrum/spectrum.min.css');
+ @import url('./prism.css');
```

Comment:
This loads prism css all the time, I wonder if we should only import this within code-related blocks (i.e. codeblock, code, tab) for optimization @timkim @BaskarMitrah what do you think?

Reply:
Yes @melissag-ensemble, It would be nice!!!. Shall I import the link into these two components only? @timkim

Reply:
Yes - we should only load it when we need it.